js/src/lib/Details/Aside.js

```diff
@@ -6,12 +6,31 @@ import Activity from './Activity';
 import Popularity from './Popularity';
 import Usage from './Usage';
 import { Owner } from '../Hit';
+import { encode, prefixURL } from '../util';
+
+const packageJSONLink = ({ githubRepo, gitHead }) => {
```

Comment:
Just use regular function syntax here
```
function packageJSONLink({ githubRepo, gitHead }) {
```

The arrow syntax is longer and doesn't really provide any added value.